packages/venia-ui/lib/components/CheckoutPage/PaymentInformation/brainTreeDropIn.js

```diff
@@ -40,40 +40,54 @@ const BraintreeDropin = props => {
     const [dropinInstance, setDropinInstance] = useState();
 
     const createDropinInstance = useCallback(async () => {
-        try {
-            const { default: dropIn } = await import('braintree-web-drop-in');
-            const dropinInstance = await dropIn.create({
-                authorization,
-                container: `#${containerId}`,
-                card: {
-                    cardholderName: {
-                        required: true
-                    },
-                    overrides: {
-                        fields: {
-                            number: {
-                                maskInput: {
-                                    // Only show last four digits on blur.
-                                    showLastFour: true
-                                }
+        const { default: dropIn } = await import('braintree-web-drop-in');
+        const dropinInstance = await dropIn.create({
+            authorization,
+            container: `#${containerId}`,
+            card: {
+                cardholderName: {
+                    required: true
+                },
+                overrides: {
+                    fields: {
+                        number: {
+                            maskInput: {
+                                // Only show last four digits on blur.
+                                showLastFour: true
                             }
                         }
                     }
                 }
-            });
-            setDropinInstance(dropinInstance);
+            }
+        });
+
+        return dropinInstance;
+    }, [containerId]);
+
+    useEffect(() => {
+        let unmounted = false;
+
+        const renderDropin = async () => {
+            const instance = await createDropinInstance();
```

Comment:
This isn't quite right. We still get an error because it's possible for the component to attempt to attach to a container which is no longer mounted.

[![Image from Gyazo](https://i.gyazo.com/f8bb7d5085707f5ac718b3675ab58dcd.png)](https://gyazo.com/f8bb7d5085707f5ac718b3675ab58dcd)

Reply:
I tried the same before committing my code and I was not able to reproduce. I tried now, and it is intermittent. Maybe I am clicking it before the initializing is done. Changing now.